source/constants/s3.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2018 iNuron NV
+#
+# This file is part of Open vStorage Open Source Edition (OSE),
+# as available from
+#
+#      http://www.openvstorage.org and
+#      http://www.openvstorage.com.
+#
+# This file is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License v3 (GNU AGPLv3)
+# as published by the Free Software Foundation, in version 3 as it comes
+# in the LICENSE.txt file of the Open vStorage OSE distribution.
+#
+# Open vStorage is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY of any kind.
+"""
+Constants involved in config management
+"""
+import os
+
+S3_BASE = '/ovs/alba/asdnodes/{0}/s3'
```

Comment:
Use the config path from here: https://github.com/openvstorage/alba-asdmanager/blob/develop/source/tools/configuration.py#L37 instead
Add the key s3 to that one
So once resolved it would be /ovs/alba/asdnodes/{}/config/main|s3